src/components/ogre/OgreSetup.cpp

```diff
@@ -188,16 +188,37 @@ void OgreSetup::createOgreSystem() {
 	mPluginLoader.loadPlugin("Codec_FreeImage");
 	mPluginLoader.loadPlugin("Plugin_ParticleFX");
 	mPluginLoader.loadPlugin("RenderSystem_GL3Plus"); //We'll use OpenGL on Windows too, to make it easier to develop
+	mPluginLoader.loadPlugin("RenderSystem_Direct3D11");
```

Comment:
Are you getting errors without these? Normally Ember should run without having to load the DirectX-plugins.